ground/src/main/java/com/google/android/ground/model/Survey.kt

```diff
@@ -24,6 +25,7 @@ data class Survey(
   val description: String,
   val jobMap: Map<String, Job>,
   val acl: Map<String, String> = mapOf(),
+  val dataSharingTerms: Survey.DataSharingTerms? = null,
```

Comment:
Asking out of curiosity, should we be mixing the proto types with our data classes?